README.md

```diff
@@ -81,15 +85,22 @@ The following are examples of valid sentences:
 Write the BNF (not EBNF) description for this language.
 
 ## A2.1
-
-«replace this with your answer»
-
+<sentance>::="the"<adjective><noun><verb>|"the"<adjective><noun><verb><adverb>
+<adjective>::= ""|"lazy"|"smelly"|"lazy"<adjective>|"smelly"<adjective>
+<noun>::="dog"|"cat"
+<verb>::="ate"|"ran"
+<adverb>::="slowly"|"noisily"
 
 ## Q2.2 (5 points)
 
 Write this grammar using EBNF with common extensions
 
 ## A2.2
+<sentance>::="the"*<adjective><noun><verb>?<adverb>
+<adjective>::="lazy"|"smelly"
+<noun>::="dog"|"cat"
+<verb>::="ate"|"ran"
+<adverb>::="slowly"|"noisily"
```

Comment:
# 5

This really confused me for a second, because the convention is to put * and ? qualifiers _after_ the term they apply to.